### PR TITLE
Handle async generator in runner-py

### PR DESCRIPTION
**Pull request by Tatarinho**
As part of this PR:
- [x] Handle async generator in runner-py
- [x] Add sample with async generator and BDD test for it
- [x] Add building runner-py image on CI

### CODE REVIEW

File: packages/python-runner/runner.py [hunk 1]
```diff
@@ -205,7 +205,9 @@ async def run_instance(self, config, args):
             self.logger.info(f'Sending PANG with {consumes}')
             send_encoded_msg(monitoring, msg_codes.PANG, consumes)
 
-        if asyncio.iscoroutine(result):
+        if isinstance(result, types.AsyncGeneratorType):
+            result = Stream.read_from(result)
```
Comment (f1ames) [inline packages/python-runner/runner.py:209]: This is basically very similar to [what we have in JS runner](https://github.com/scramjetorg/transform-hub/blob/devel/packages/runner/src/runner.ts#L476-L480) so it should align runner behaviors :+1:

File: python/reference-apps/python-gen-async/main.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+async def save_entry_to_db(id: str) -> None:
```
Comment (f1ames) [inline python/reference-apps/python-gen-async/main.py:1]: Maybe a test which run this sequence? Or you think it's not necessary?

**Review by f1ames — DISMISSED** [review]
Looks good :+1: 

Just one suggestion regarding tests (see comment).

File: python/reference-apps/python-gen-async/main.py (hunk 2 above)
Comment (Tatarinho) [reply python/reference-apps/python-gen-async/main.py:1]: I can write one simple test I think

**Review by MichalCz — APPROVED** [review]

**Pull request merged**